app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @email %>!</p>
+
+<% if @resource.try(:unconfirmed_email?) %>
+  <p>We're contacting you to notify you that your email is being changed to <%= @resource.unconfirmed_email %>.</p>
```

Comment:
I18n đi anh ơi

Reply:
đã chỉnh I18n cho toàn bộ views của devise